frontend/apps/pablo/components/Organisms/auction/AuctionPriceChart.tsx

```diff
@@ -7,6 +7,57 @@ import { MockedAsset } from "@/store/assets/assets.types";
 
 const NoSSRChart = dynamic(() => import("react-apexcharts"), { ssr: false });
 
+type PriceChartLabelProps = {
+  priceSeries: [number, number][];
+  predictedPriceSeries: [number, number][];
+  assetSymbol?: string;
+};
+
+const PriceChartLabels = ({
+  priceSeries,
+  predictedPriceSeries,
+  assetSymbol,
+}: PriceChartLabelProps) => {
+  if (priceSeries.length > 0 && predictedPriceSeries.length > 0) {
+    return (
+      <>
+        <FiberManualRecordIcon color="primary" />
+        <Typography variant="body2" pl={1} pr={2}>
+          {assetSymbol}
+        </Typography>
+        <FiberManualRecordIcon color="inherit" />
+        <Typography variant="body2" pl={1} whiteSpace="nowrap">
+          {assetSymbol} predicted price (without new buyers)
+        </Typography>
+      </>
+    );
+  }
+
+  if (priceSeries.length <= 0 || predictedPriceSeries.length <= 0) {
```

Comment:
(question)

Why not use `priceSeries.length === 0` or the simpler: `Array.isArray(priceSeries) && !priceSeries.length`?

Reply:
yes for sure, updated